muxers/mplex/src/io.rs

```diff
@@ -40,11 +41,16 @@ pub struct Multiplexed<C> {
     io: Fuse<Framed<C, Codec>>,
     /// The configuration.
     config: MplexConfig,
-    /// Buffer of received frames that have not yet been consumed.
-    buffer: Vec<Frame<RemoteStreamId>>,
+    /// The buffer of new inbound substreams that have not yet
+    /// been drained by `poll_next_stream`.
+    open_buffer: VecDeque<LocalStreamId>,
     /// Whether a flush is pending due to one or more new outbound
     /// `Open` frames, before reading frames can proceed.
     pending_flush_open: bool,
+    /// The stream that currently blocks reading for all streams
+    /// due to a full buffer, if any. Only applicable for use
+    /// with [`MaxBufferBehaviour::Block`].
+    blocked_stream: Option<LocalStreamId>,
```

Comment:
```suggestion
    blocking_stream: Option<LocalStreamId>,
```

Wouldn't `blocking` be more descriptive, given that this stream is blocking other streams which are thus blocked?

Reply:
I don't think I could infer the behaviour w.r.t. all substreams from either name and would always need to look at the comment, but I'm fine with the name change.

Reply:
Done in https://github.com/libp2p/rust-libp2p/pull/1784/commits/d689611611763c2b844afce13aab500204ab2d6f.